packages/wallet/src/services/UtxoTracker.ts

```diff
@@ -40,28 +41,34 @@ export const createUtxoTracker = (
   {
     utxoSource$ = new PersistentCollectionTrackerSubject<Cardano.Utxo>(
       () => createUtxoProvider(walletProvider, addresses$, tipBlockHeight$, retryBackoffConfig),
-      store
-    )
+      store.utxo
+    ),
+    reservedUtxoSource$ = new TrackerSubject(store.reservedUtxo.getAll())
```

Comment:
`getAll()` will emit 0 or 1 times and complete. You need to `concat` it with something that emits when you `setReserved()`

Reply:
I'm a bit lost here, I don't understand what I need to do to stop it from completing. 
> Concat it with another Subject, calling .next() on setReserved

If I go with this option, what can I concat it with? 

and if I use `PersistentCollectionTrackerSubject`, what should I use as the `source` if I only have the store? You mentioned a refactor, but I don't know what I should do besides adding
```ts
next(value: T[]): void {
  this.store.setAll(value);
}
```